daemon/list.go

```diff
@@ -516,6 +517,21 @@ func (daemon *Daemon) filterVolumes(vols []volume.Volume, filter filters.Args) (
 				continue
 			}
 		}
+		if filter.Include("label") {
+			v, err := daemon.volumes.Get(vol.Name())
```

Comment:
@calavera  #22077  did some change on volume files, I'd like to wait for that to merge, in case waste our rebase effort for that.